lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -249,6 +233,19 @@ sub _resolve_reference {
 sub __errors__ {
     my $self   = shift;
     my @errors = $self->SUPER::__errors__;
+
+    for my $model (qw(wgs_model exome_model)) {
+        if ($self->$model) {
+            unless ($self->$model->isa('Genome::Model::SomaticVariation') || $self->$model->isa('Genome::Model::SomaticValidation')) {
```

Comment:
That seems unreliable, since it's dependent on a build existing to pass.

A few options:
* Implement `Genome::Model::SomaticInterface` (some of the accessors like `individual` on the build class might end up delegating there)
* Make a `build_class` like `Genome::Model::Build::model_class` and then it's `$self->$model->build_class->isa('Genome::Model::Build::SomaticInterface')`